content/en/blog/2023/outreachy-how-why-sakshi.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Outreachy - How, When & Why?
+linkTitle: Introduction to Outreachy 2023 # Mandatory, make sure that your short title.
```

Comment:
```suggestion
linkTitle: Introduction to Outreachy 2023 # Mandatory, make sure that your short title.
```

The linkTitle should be short and only a few words, like "Outreachy Internship" or "Outreachy Report", etc.